File: decompiled/3.17.1/be/ehealth/businessconnector/mycarenet/memberdata/builders/RequestObjectBuilder.java
package be.ehealth.businessconnector.mycarenet.memberdata.builders;

import be.ehealth.business.mycarenetdomaincommons.domain.InputReference;
import be.ehealth.businessconnector.mycarenet.memberdata.exception.MemberDataBusinessConnectorException;
import be.ehealth.technicalconnector.exception.TechnicalConnectorException;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest;
import oasis.names.tc.saml._2_0.protocol.AttributeQuery;

public interface RequestObjectBuilder {
   MemberDataConsultationRequest buildConsultationRequest(boolean var1, InputReference var2, AttributeQuery var3) throws TechnicalConnectorException, MemberDataBusinessConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/memberdata/builders/RequestObjectBuilder.kt
package org.taktik.connector.business.memberdata.builders

import org.taktik.connector.technical.exception.TechnicalConnectorException
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest
import oasis.names.tc.saml._2_0.protocol.AttributeQuery
import org.taktik.connector.business.memberdata.exception.MemberDataBusinessConnectorException
import org.taktik.connector.business.mycarenetdomaincommons.domain.InputReference

interface RequestObjectBuilder {
    @Throws(TechnicalConnectorException::class, MemberDataBusinessConnectorException::class)
    fun buildConsultationRequest(var1: Boolean, var2: InputReference, var3: AttributeQuery): MemberDataConsultationRequest
}
